Source/Core/Updater/UI.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <Windows.h>
+
+#include "Common/Flag.h"
+
+namespace UI
+{
+void Loop();
```

Comment:
This is kind of ambiguously named.

Reply:
It's the UI Loop that handles all window functions. I don't know what else to call it.
Feel free to suggest something though!

Reply:
MessageLoop()

Reply:
Fine by me!